system-apps/admin-notification-app/webClient/src/app/adminnotification-component.ts

```diff
@@ -50,12 +50,23 @@ export class AdminNotificationComponent {
 
   sendRest(title: string, message: string): void {
     let notification = new ZoweNotification(title, message, 1, "org.zowe.zlux.bootstrap")
+    let element = (<HTMLImageElement>document.getElementsByClassName('response')[0]);
```

Comment:
'response' is a pretty generic class name so this is waiting to get broken by something via an element array with length > 1